tune_sklearn/tune_basesearch.py

```diff
@@ -67,13 +70,45 @@ def resolve_early_stopping(early_stopping, max_iters):
                         f"or tune scheduler. Got {type(early_stopping)}.")
 
 
+def resolve_loggers(loggers):
+    if loggers is None:
+        return None
+
+    if not isinstance(loggers, list):
+        raise TypeError("`loggers` must be a list of str or tune loggers.")
+
+    init_loggers = {JsonLogger, CSVLogger}
+    for log in loggers:
+        if isinstance(log, str):
+            if log == "tensorboard":
+                init_loggers.add(TBXLogger)
+            elif log == "csv":
+                init_loggers.add(CSVLogger)
+            elif log == "mlflow":
+                init_loggers.add(MLFLowLogger)
+            elif log == "json":
+                init_loggers.add(JsonLogger)
+            else:
+                raise ValueError("{} is not a defined logger. "
+                                 "Check the list of available loggers."
```

Comment:
Can you print the list of defined loggers? You could pass it in, or set it as a global variable